### ISSUE TITLE
Documentation: updating screenshots

### ISSUE THREAD

**Issue by vindarel** [herth/ltk#4]
Hi,

If I wanted to update and add screenshots, could I do it ? Is ltk's website on a public repository open to patches or pull requests ? 

The linux screenshots are really outdated.

Regards

**Comment by chuchana** [comment]
I agree that the documentation could use some updates and would like to correct the flaws that I find. E.g. the ttk widgets do not support many of the configuration options. That's fine if you know it, but this piece of information is missing in the documentation.

By the way: the documentation currently linked to on [the website](http://www.peter-herth.de/ltk/) is not the latest one available. The newest version I found is [here](http://www.peter-herth.de/ltk/ltkdoc-single/ltkdoc.html). I have contacted the author about it, so I assume he will update the link when he finds the time.
